src/SDKs/Peering/Management.Peering.Tests/Peering.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
Please remove this `PropertyGroup`. Refer this and remove any additional `PropertyGroup`/`ItemGroup`